dockers/docker-orchagent/start.sh

```diff
@@ -5,7 +5,11 @@ function start_app {
     portsyncd $PORTSYNCD_ARGS &
     intfsyncd &
     neighsyncd &
-    swssconfig &
+    for file in $SWSSCONFIG_ARGS
+    do
+      swssconfig /etc/swss/config.d/$file
+      sleep 1
```

Comment:
Not got the idea. Do you mean code changes in swssconfig? What is the problem here?